esphome/components/lilygo_t5_47/display/lilygo_t5_47_display.h

```diff
@@ -40,7 +40,9 @@ class LilygoT547Display : public PollingComponent, public display::DisplayBuffer
   void flush_screen_changes();
   void power_on();
   void power_off();
-
+  
+  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_BINARY; }
```

Comment:
I don't know whether your implementation supports grayscale or only binary display; if grayscale is supported, you should use that instead:
```suggestion
  display::DisplayType get_display_type() override { return display::DisplayType::DISPLAY_TYPE_GRAYSCALE; }
```
(or if both are supported, you'll need an if or a ternary operator to decide at runtime).

Reply:
It's binary only - thanks!